ftp/src/main/scala/akka/stream/alpakka/ftp/Ftp.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.ftp
+
+import org.apache.commons.net.ftp.FTPClient
+import scala.collection.immutable
+import scala.util.Try
+
+/**
+ * @author Juan José Vázquez Delgado
```

Comment:
please remove, explained in https://github.com/akka/alpakka/blob/master/CONTRIBUTING.md#pull-request-requirements